content/docs/en/elements/layouts/grid-layout.md

```diff
@@ -51,23 +55,43 @@ The GridLayout is a layout that arranges its child elements in a table structure
 ```
 <img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout_fixed_auto.svg" />
 
+#### Grid layout with complex sizing and merged cells
+
+The following example creates a complex grid with responsive design, mixed width and height settings, and some merged cells.
+
+```html
+<GridLayout columns="40, auto, *" rows="40, auto, *" backgroundColor="#3c495e">
+  <Label text="0,0" row="0" col="0" backgroundColor="#43b883"/>
+  <Label text="0,1" row="0" col="1" colSpan="2" backgroundColor="#1c6b48"/>
+  <Label text="1,0" row="1" col="0" rowSpan="2" backgroundColor="#289062"/>
+  <Label text="1,1" row="1" col="1" backgroundColor="#43b883"/>
+  <Label text="1,2" row="1" col="2" backgroundColor="#289062"/>
+  <Label text="2,1" row="2" col="1" backgroundColor="#1c6b48"/>
+  <Label text="2,2" row="2" col="2" backgroundColor="#43b883"/>
+</GridLayout>
+```
+<img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout.svg" />
 
 ## Props
 
-| name | type | description |
+| Name | Type | Description |
 |------|------|-------------|
-`columns` | `String` | A string value representing column widths delimited with commas. Column widths can be either an absolute number, `auto` or `*`. A number indicates an absolute column width, `auto` makes the column as wide as its widest child, and `*` makes the column occupy all available horizontal space.
```

Comment:
I think we should leave this part in:

> `auto` makes the column as wide as its widest child, and `*` makes the column occupy all available horizontal space

Reply:
Applied. Appears as change several lines below.